p2panda-auth/src/group/crdt/mod.rs

```diff
@@ -165,7 +180,14 @@ where
         group_id: ID,
         members: &mut HashMap<ID, Access<C>>,
         root_access: Option<Access<C>>,
+        mut depth: u32,
     ) {
+        // If we reached max nesting depth exit from the traversal.
+        if depth == MAX_NESTED_DEPTH {
```

Comment:
Is there any point at which we communicate "upstream" that we've hit this limit? Or would you imagine an application author needing to ensure that they didn't allow nesting beyond this point? Currently the code returns silently.

Reply:
This I'm not sure how to handle, I could look into introducing an error type, but intuitively I feel that might introduce errors to the API in a way that makes it a bit ugly. I thought that communicating the limit and behaviour might be enough, again considering that it's mostly a fail-safe for group cycles.

Reply:
I've made this into an error as I'd like to consider it more after merging this PR https://github.com/p2panda/p2panda/issues/799

Thanks for raising this issue!